src/utils/constants.js

```diff
@@ -1 +1,49 @@
-export const API_BASE_URL = 'https://wizeline-academy.cdn.prismic.io/api/v2';
\ No newline at end of file
+export const API_BASE_URL = 'https://wizeline-academy.cdn.prismic.io/api/v2';
```

Comment:
Esta constante podría definirse como variable de entorno, si no me equivoco en netlify puedes agregarlas y aquí en el codiogo seria algo como REACT_APP_API_BASE_URL https://create-react-app.dev/docs/adding-custom-environment-variables/ asi ya estas que podrían a lo mejor ser información sensible no estarian aqui

Reply:
Will change it to env vars.